install/ontology/lti.rdf

```diff
@@ -161,15 +164,17 @@
     <rdfs:domain rdf:resource="http://www.tao.lu/Ontologies/TAOLTI.rdf#LTIProvider"/>
     <rdfs:range rdf:resource="http://www.w3.org/2000/01/rdf-schema#Literal"/>
     <widget:widget rdf:resource="http://www.tao.lu/datatypes/WidgetDefinitions.rdf#TextBox"/>
+    <tao:TAOGUIOrder><![CDATA[200]]></tao:TAOGUIOrder>
   </rdf:Description>
 
   <rdf:Description rdf:about="http://www.tao.lu/Ontologies/TAOLTI.rdf#toolClientId">
     <rdf:type rdf:resource="http://www.w3.org/1999/02/22-rdf-syntax-ns#Property"/>
-    <rdfs:label xml:lang="en-US"><![CDATA[Tool client id]]></rdfs:label>
-    <rdfs:comment xml:lang="en-US"><![CDATA[Tool client id]]></rdfs:comment>
+    <rdfs:label xml:lang="en-US"><![CDATA[Client id]]></rdfs:label>
+    <rdfs:comment xml:lang="en-US"><![CDATA[Client id]]></rdfs:comment>
```

Comment:
`Tool client Id` would be better, cause we are using prefix `Tool` for all fields.